SPECS/openssl/openssl.spec

```diff
@@ -43,6 +41,9 @@ Patch20:        openssl-1.1.1-jitterentropy.patch
 Patch21:        openssl-1.1.1-drbg-seed.patch
 Patch22:        CVE-2021-3711.patch
 Patch23:        CVE-2021-3712.patch
+Patch24:        openssl-1.1.1-fips-SymCrypt.patch
+%define __requires_exclude ^/(bin|usr/bin).*$
```

Comment:
nit - Apologies that the linter complained about this but please keep these defines at the top of the spec file.